sgl-kernel/setup_rocm.py

```diff
@@ -71,7 +71,7 @@ def _get_version():
         },
         libraries=libraries,
         extra_link_args=extra_link_args,
-        py_limited_api=True,
+        py_limited_api=False,
```

Comment:
Is it compatible with `options={"bdist_wheel": {"py_limited_api": "cp39"}}`

Reply:
No issue is seen for dist_wheel

Reply:
In previous implementations, compiled in the environment of Python 3.9, whl can also be used in Python 3.10. Can you still guarantee this after your changes? Thanks!

Reply:
Our docker env is upgrade to cp12 a while ago, still worked.